internal/saucecloud/cypress.go

```diff
@@ -28,7 +30,14 @@ func (r *CypressRunner) RunProject() (int, error) {
 		files = append(files, r.Project.Cypress.EnvFile)
 	}
 
-	fileID, err := r.archiveAndUpload(r.Project, files)
+	if r.Project.RootDir != "" {
+		files = append(files, r.Project.RootDir)
+	}
+
+	// path to the cypress.json and .sauceignore
+	configDir := filepath.Dir(r.Project.Cypress.ConfigFile)
```

Comment:
I though that `.sauceignore` would be in `rootDir` ?
Should it be ignore with the following folder:
```
/path/to/my/project
+- .sauceignore
+- tests
   +- cypress
      +- cypress.json
      +- cypress
         +- integration
            +- example.test.js

```

and following config:
```
[...]
rootDir: /path/to/my/project
cypress:
  config: tests/cypress/cypress.json
[...]
```

Reply:
`rootDir` is an optional field. A user can skip this field and still use `.sauceignore`.  When a user runs `saucectl new` we will create `.sauceignore` on the same level as `cypress.json`.

Reply:
1. I'd suggest to simply use the working directory of saucectl. That behavior is consistent across all frameworks and whether or not you have a root dir defined (e.g. mobile won't have root dirs).
2. Have this parameter be configurable via an arg in the CLI.

Reply:
@alexplischke what is the best suitable name for this CLI arg in this case? `sauceignore-path` ?

Reply:
`--ignore` or `--sauceignore` would do :)

Reply:
Should we add a config field for this file too ?

Reply:
I wouldn't do that. In case we have `sauceignore` as config we need to support 3 different cases. First is the working directory of `saucectl`, the second is CLI arg and finally `config.sauceignore`. I would leave working directory and CLI param.

Reply:
I added support of `sauceignore` CLI arg and in case if it's empty just use the working directory.